tests/unit/features/overlay/test_parts.py

```diff
@@ -407,17 +407,27 @@ def test_part_unmarshal_type_error(self):
         with pytest.raises(errors.PartSpecificationError) as raised:
             Part("foo", {"plugin": []})
         assert raised.value.part_name == "foo"
-        assert raised.value.message == "- str type expected in field 'plugin'"
+        assert (
+            raised.value.message == "- Input should be a valid string in field 'plugin'"
+        )
+
+    @pytest.mark.parametrize("fileset", ["stage", "prime"])
+    def test_relative_path_validation_slash(self, fileset):
+        with pytest.raises(errors.PartSpecificationError) as raised:
+            Part("foo", {fileset: ["bar", "/baz", "x"]})
+        assert raised.value.part_name == "foo"
+        assert raised.value.message == (
+            "- Value error, '/baz' must be a relative path (cannot start with "
```

Comment:
If possible I think we should lose this `Value error, ` prefix

Reply:
Unfortunately, it is something coming from pydantic2.